### PR TITLE
feat: tree view style

### PR DESCRIPTION
**Pull request by hadaromer**
- added style to tree view
- added version selector instead of another tree leaf
- fixed support to version query

### CODE REVIEW

File: src/components/group/selector/TreeLevel.js [hunk 1]
```diff
@@ -32,38 +34,40 @@ export default function TreeView(props) {
...
   };
 
   return (
     nodes && (
-      <TreeItem
+      <TreeNode
         key={path}
         nodeId={path}
-        label={path.substring(path.lastIndexOf('/') + 1)}
+        labelText={path.substring(path.lastIndexOf('/') + 1)}
```
Comment (asafMasa) [inline src/components/group/selector/TreeLevel.js]: refactor to util function

File: src/components/group/selector/TreeNodeItem.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  }, []);
+
+  const getFromServer = async function () {
+    // Get items from the server.
+    try {
+      const res = await itemRequest(path, name);
+      setItem(res);
+      setVersion(res.latest.version);
+    } catch {
+      enqueueSnackbar(ERROR_CONTACTING_SERVER.message, {
```
Comment (asafMasa) [inline src/components/group/selector/TreeNodeItem.js:25]: logger

File: src/components/group/selector/VersionSelector.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Select, MenuItem } from '@material-ui/core/';
+
+export default function VersionSelector(props) {
+  const { version, versions, setVersion, clickOnList } = props;
+
+  const handleChange = function (e) {
```
Comment (asafMasa) [inline src/components/group/selector/VersionSelector.js]: () =>

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/components/group/selector/TreeNodeItem.js (hunk 2 above)
Comment (shimoncohen) [reply src/components/group/selector/TreeNodeItem.js:25]: Logging happens in request function.

**Pull request merged**